app/src/main/java/gitlin/kothub/utilities/JsonUtilities.kt

```diff
@@ -23,4 +23,7 @@ fun <T, V> JSONArray.map(mapper: (T) -> V): MutableList<V> {
     return (0..length() - 1).mapTo(arrayListOf<V>()) { mapper(get(it) as T) }
 }
 
+operator fun JSONArray.iterator(): Iterator<JSONObject>
```

Comment:
It's used here: https://github.com/kothub/kothub/blob/dbc71c61f45be653944327aa98f95de90a62c3f8/app/src/main/java/gitlin/kothub/github/api/Viewer.kt#L69